todo.js

```diff
@@ -124,7 +114,53 @@ function updateTable() {
 
 // Initialize table with current month
 window.onload = function () {
-  //const currentMonth = new Date().getMonth();
-  //document.getElementById('monthSelector').value = currentMonth;
-  createTable();
+  createTable(state.todoList);
+};
+
+//Filter stored tomatoes by id, year and month
+const currentTomatoes = (id) => {
```

Comment:
1. function names are better if they start with a **verb**, like "get", "show", "filter" or even "do", that way we can tell it's a function right away when we see it referenced anywhere in the code
2. it looks like this function returns **just one** todo? if so, "Tomatoes" -> "Tomato"
```suggestion
const getCurrentTomato = (id) => {
```

Reply:
I delete it